pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -523,7 +523,7 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 			tr("Tries to detect when a game is drawing its own color palette and then renders it in software, instead of on the GPU."));
 
 		dialog->registerWidgetHelp(m_ui.gpuTargetCLUTMode, tr("GPU Target CLUT"), tr("Disabled"), 
-			tr("Tries to detect when a game is drawing its own color palette and then renders it on the GPU with special handling"));
+			tr("Tries to detect when a game is drawing its own color palette and then renders it on the GPU with special handling."));
```

Comment:
I think using the infinitive is better here, even though it's a description:
`Try to detect when a game is drawing its own color palette, and render it on the GPU with special handling.`